draft-ietf-rtcweb-jsep.xml

```diff
@@ -3222,6 +3181,49 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           below.</t>
         </list></t>
       </section>
+      <section title="Processing a Rollback" anchor="sec.processing-a-rollback">
+        <t>A rollback may be performed if the PeerConnection is in any
+        state except for "stable". This means that both offers and provisional
+        answers can be rolled back. If a rollback is attempted in the "stable"
+        state, processing MUST stop and an error MUST be returned.The effect
+        MUST be the same regardless of whether setLocalDescription or
+        setRemoteDescription is called.</t>
+        
+        <t>In order to process rollback, a JSEP implementation
```

Comment:
The future tense used in this section is inconsistent with the surrounding sections. Suggest changing "will be processed" to "is processed" below.